experimenter/experimenter/nimbus_ui_new/filtersets.py

```diff
@@ -202,7 +202,10 @@ class NimbusExperimentFilter(django_filters.FilterSet):
     )
     takeaways = django_filters.MultipleChoiceFilter(
         method="filter_takeaways",
-        choices=NimbusExperiment.Takeaways.choices,
+        choices=(
+            NimbusExperiment.Takeaways.choices
+            + NimbusExperiment.ConclusionRecommendation.choices
+        ),
```

Comment:
Oh yes, but also I was gonna say there's a nicer way to say this now

```py
[
    *NimbusExperiment.Takeaways.choices,
    *NimbusExperiment.ConclusionRecommendation.choices,
]
```

since using math operands on collections technically functions but is a bit of an old/awkward way to say that.